package.json

```diff
@@ -40,6 +41,8 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
+    "@adonisjs/generic-exceptions": "^3.0.1",
```

Comment:
We should be using `@poppinss/utils` and use the `Exception` class exported by it. The `@adonisjs/generic-exceptions` is depreciated